instabug/assets/app-builder-blueprints/instabug-sessions-explorer.json

```
@@ -1,2177 +1,1969 @@
+                                                        },
+                                                        "type": "vegaChart"
+                                                      }
+                                                    ],
+                                                    "isVisible": "true",
+                                                    "layout": {
+                                                      "default": {
+                                                        "height": 36,
+                                                        "width": 6,
+                                                        "x": 0,
+                                                        "y": 0
+                                                      }
+                                                    }
+                                                  },
+                                                  "type": "gridCell"
+                                                },
+                                                {
+                                                  "events": [],
+                                                  "name": "gridCell11",
+                                                  "properties": {
+                                                    "children": [
+                                                      {
+                                                        "events": [],
+                                                        "name": "cfr_overtime_chart",
+                                                        "properties": {
+                                                          "errorDescription": "",
+                                                          "errorMessage": "",
+                                                          "hasBorders": false,
+                                                          "isLoading": "${cfr_overtime_chart_request?.isLoading}",
+                                                          "isVisible": true,
+                                                          "title": "Crash-free rates Overtime",
```

Ignore this suggestion if "overtime" is intentional!

```suggestion
                                                          "title": "Crash-free rates over time",
```